Backpack/src/main/java/net/skyscanner/backpack/calendar/view/CalendarView.kt

```diff
@@ -61,6 +63,21 @@ internal class CalendarView constructor(context: Context, attr: AttributeSet?) :
     setUpListView()
   }
 
+  private val measureAndLayout = Runnable {
+    measure(
+      View.MeasureSpec.makeMeasureSpec(width, View.MeasureSpec.EXACTLY),
+      View.MeasureSpec.makeMeasureSpec(height, View.MeasureSpec.UNSPECIFIED))
+    layout(left, top, right, bottom)
+  }
+
+  override fun requestLayout() {
+    super.requestLayout()
+
+    // This is required for the bridged component to render correctly
+    // based on: https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/ReactAndroid/src/main/java/com/facebook/react/views/picker/ReactPicker.java#L75
+    this.post(measureAndLayout)
```

Comment:
Is this native component going to be used in React Native code as well? Isn't there a React Native calendar implementation already?

Reply:
No, we don't have it for React Native, those changes are exactly for that purpose. We are bridging this component to react native here https://github.com/Skyscanner/backpack-react-native/pull/34/files

Reply:
Good to know!